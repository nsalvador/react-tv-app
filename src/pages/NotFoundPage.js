import React from 'react';
import { Link } from 'react-router-dom';

import Layout from '../components/Layout';

const NotFoundPage = () => (
	<Layout>
		404 - <Link to="/dashboard">Go Home</Link>
	</Layout>
);

export default NotFoundPage;
