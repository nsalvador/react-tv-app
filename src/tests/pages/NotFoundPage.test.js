import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../pages/NotFoundPage';

it('should render NotFoundPage correctly', () => {
	const wrapper = shallow(<NotFoundPage />);
	expect(wrapper).toMatchSnapshot();
});
