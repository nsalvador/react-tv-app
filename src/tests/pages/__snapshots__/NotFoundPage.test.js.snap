// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<div>
  <Layout>
    404 - 
    <Link
      to="/dashboard"
    >
      Go Home
    </Link>
  </Layout>
</div>
`;
